### ISSUE TITLE
Inconsistent and 'forgotten' theme settings 

### ISSUE THREAD

**Issue by Al-Q** [mate-desktop/mate-themes#199]
#### Expected behaviour
(Customized) themes remain consistent between logins; theme settings are remembered between logins. 

#### Actual behaviour
I created a modified theme so that I can easily grab window borders to resize with the mouse and still have a nice looking theme. Specifically Ambiant-MATE with Atlanta borders. 

Almost always, after a logout/login (reboot) sequence, text in tabs of *remembered* applications (Firefox and Thunderbird) is a very similar colour to the background and therefore unreadable. In other words, the colours chosen by me are not the colours used by the desktop after a reboot/logout-login. 

Sometimes I can go to 'Appearance Preferences', choose another theme, then switch back to my customized theme, and the settings are correct. Yesterday, that worked on Thunderbird but not on Firefox. Today, even switching to a 'standard' non-modified theme and rebooting doesn't help, the tab bug is still there -- it's getting worse. Sometimes it will work on a foreground window, but not always. Even switching to a standard/non-modified theme doesn't work -- I still have dark-on-dark tab colours. 

#### Steps to reproduce the behaviour
(This is an unstable bug: most often it's there, sometimes it's not; sometimes the switching fix works, mostly it doesn't.)  
 
In Control Center > Startup Applications > Options , Select 'Automatically remember running applications when logging out' 
In Appearance Preferences, customize-and-save/choose a theme (eg Ambiant-MATE with Atlanta borders). Switch to that theme.  
Log out. Log in. 

#### MATE general version
MATE Desktop Environment 1.12.1 

#### Package version
`apt-cache showpkg mate-desktop`
```
Package: mate-desktop
Versions: 
1.12.1-1 (/var/lib/apt/lists/gb.archive.ubuntu.com_ubuntu_dists_xenial_universe_binary-amd64_Packages) (/var/lib/dpkg/status)
 Description Language: 
                 File: /var/lib/apt/lists/gb.archive.ubuntu.com_ubuntu_dists_xenial_universe_binary-amd64_Packages
                  MD5: 00a366bf9bf6902235e7787f948836fd
 Description Language: 
                 File: /var/lib/apt/lists/gb.archive.ubuntu.com_ubuntu_dists_xenial_universe_binary-i386_Packages
                  MD5: 00a366bf9bf6902235e7787f948836fd
 Description Language: en
                 File: /var/lib/apt/lists/gb.archive.ubuntu.com_ubuntu_dists_xenial_universe_i18n_Translation-en
                  MD5: 00a366bf9bf6902235e7787f948836fd

Reverse Depends: 
  mate-desktop-dbg,mate-desktop 1.12.1-1
  mate-media-pulse,mate-desktop 1.5.0
  caja,mate-desktop
  mate-desktop:i386,mate-desktop
  mate-panel,mate-desktop
  mate-desktop-environment-core,mate-desktop 1.12.0
  mate-control-center,mate-desktop 1.11.0
  mate-desktop-common,mate-desktop 1.1.1-1
  mate-desktop-common,mate-desktop 1.1.1-1
  cairo-dock-core,mate-desktop
  caja,mate-desktop
Dependencies: 
1.12.1-1 - hicolor-icon-theme (0 (null)) libmate-desktop-2-17 (2 1.10.0) mate-desktop-common (5 1.12.1-1) mate-user-guide (0 (null)) libc6 (2 2.4) libglib2.0-0 (2 2.28.0) libgtk2.0-0 (2 2.14.0) python (0 (null)) mate-desktop-gnome (0 (null)) mate-desktop-gnome (0 (null)) mate-desktop:i386 (32 (null)) 
Provides: 
1.12.1-1 - mate-desktop:i386 (= 1.12.1-1) 
Reverse Provides: 
mate-desktop:i386 1.12.1-1 (= 1.12.1-1)
```

`dpkg -l libgtk2.0-0`
```
ii  libgtk2.0-0:am 2.24.30-1ubu amd64        GTK+ graphical user interface lib
```

#### Linux Distribution
Ubuntu  MATE 16.04

#### Link to downstream report of your Distribution
? 

#### Comments 
My apologies for the lack of clarity -- the problem is hard to pin down. 

For me this is the only problem -- Everything else about MATE is excellent.

**Comment by lukefromdc** [comment]
I just found that this does not occur in MATE 1.19 (git master) on Debian Unstable. I created a test theme with window borders changed, set up session saving, started a new session, started firefox, changed to that theme, and on logging out and back in Firefox (version 57) came back up with the modified theme showing. Thunderbird I don't have installed so could not test that.

 MATE 1.12 is a very old version, so this bug may be a thing of the past. Can anyone else duplicate this with current versions?
